services/backup-handler/internal/handler/main.go

```diff
@@ -224,7 +240,7 @@ func ProcessBackups(backupData Backups, backupsEnv api.Environment) []Webhook {
 
 func failOnError(err error, msg string) {
 	if err != nil {
-		log.Printf("rabbit failure, error is %s:", err.Error())
+		log.Printf("rabbit failure, error is %s/%s:", msg, err.Error())
```

Comment:
what's the second %s for?

Reply:
Nah its used. Since the `failOnError` accepts a msg input, there are parts where `failOnError(err, "Failed to connect to RabbitMQ")` and other msgs are used. So this change is just exposing them now so it will be
```
Failed to connect to RabbitMQ/couldn't connect due to whatever reason
Could not declare queue/queue whatever doesnt exist
```